### DISCUSSION TITLE
Hows the new extensions manager working?

### DISCUSSION THREAD

**Discussion by Lumpsack** [vladmandic/sdnext#942] [Q&A]
Hi,
I'm not having much luck with the new extensions - it looks like it'll do something when I ask it to update all, but it does a counter then nothing looks to be updated.
If I try to update one individually I see it try in the console, but nothing, also I tried doing an uninstall via the interface and reloading, but its still there.
![image](https://github.com/vladmandic/automatic/assets/127429289/8aeff327-2b52-42bb-b3e9-46c9169fd34e)

**Comment by vladmandic** [comment]
depending on extension, they can be "stuck" in memory and there is no way to unload them from python - if that's the case, only solution is to restart server from command line and reload browser.

**Comment by Lumpsack** [comment]
Thanks, I just tried closing the server and reloading, then immediately tried updating an extension, same as before sadly, for info I'm on Version: 89d0a841 Sun May 14 08:35:39 2023 -0400.
Anything I should check?  The D:VladautomaticextensionsEasyNegative looks sort of odd, the drive is correct, but if there is supposed to be a path its not right, I think it should be : D:\Vlad\automatic\extensions
![image](https://github.com/vladmandic/automatic/assets/127429289/13e36218-b8a8-4f54-8ce5-db54a408dba1)

**Comment by vladmandic** [comment]
found an issue and fixed it.
